pkg_resources/extern/__init__.py

```diff
@@ -60,6 +54,13 @@ def create_module(self, spec):
     def exec_module(self, module):
         pass
 
+    def find_spec(self, fullname, path=None, target=None):
+        """Return a module spec for vendored names."""
+        return (
+            importlib.machinery.ModuleSpec(fullname, self)
```

Comment:
It looks like this could (should?) be https://docs.python.org/3/library/importlib.html#importlib.util.spec_from_loader.